core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -204,22 +218,44 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
                  scroll-on-click
                  is-dropdown-open="activeExplorationId === exploration.id"
                  is-private="exploration.status === 'private'"
-                 ng-click="openExploration(exploration.status, exploration.id)">
-          <div class="title-section"
-               ng-class="{'oppia-exp-arrow': activeExplorationId === exploration.id}"
-               style="background-color: <[exploration.thumbnail_bg_color]>;">
-            <img class="thumbnail-image"
-                 ng-src="<[exploration.thumbnail_icon_url]>"
-                 ng-hide="activeExplorationIdOnMobile(exploration.id)">
-            <span class="thumbnail-image caret-thumb fa fa-caret-up"
-                  ng-show="activeExplorationIdOnMobile(exploration.id)">
-            </span>
-            <h2 class="activity-title protractor-test-exp-summary-tile-title">
-              <span><[exploration.title || 'Untitled' | truncate:40]></span>
-            </h2>
+                 ng-click="getStatsForPublishedExp(exploration.status, exploration.id)">
+          <div ng-if="exploration.status !== 'private'">
+            <div class="title-section"
+                 ng-class="{'oppia-exp-arrow': activeExplorationId === exploration.id}"
+                 style="background-color: <[exploration.thumbnail_bg_color]>;">
+              <img class="thumbnail-image"
+                   ng-src="<[exploration.thumbnail_icon_url]>"
+                   ng-hide="activeExplorationIdOnMobile(exploration.id)">
+              <span class="thumbnail-image caret-thumb fa fa-caret-up"
+                    ng-show="activeExplorationIdOnMobile(exploration.id)">
+              </span>
+              <h2 class="activity-title protractor-test-exp-summary-tile-title">
+                <span><[exploration.title || 'Untitled' | truncate:40]></span>
+              </h2>
+            </div>
+            <div class="mask-wrap">
+              <div class="title-section-mask"></div>
+            </div>
           </div>
-          <div class="mask-wrap">
-            <div class="title-section-mask"></div>
+          <div ng-if="exploration.status === 'private'">
+            <a ng-href="/create/<[exploration.id]>">
+              <div class="title-section"
```

Comment:
This and the block of html above this is exactly the same, with just the difference that this one is wrapped inside an anchor tag. Do you think you'll be able to do this programmatically some way ? (I would suggest writing a simple directive that wraps an element around another based on the value of `exploration.status` being passed to it!) It'll make the code far cleaner and less redundant IMO.